lib/bot.js

```diff
@@ -119,9 +139,38 @@ class Bot {
   }
 
   /**
-   * Listens for new chat messages on all channels. The `onMessage` function is called for every message your bot receives.
-   * @param onMessage
-   * @memberof Bot
+   * This function will put your bot into insane mode, where it reads
+   * everything it can and every new message it finds it will pass to you, so
+   * you can do what you want with it. For example, if you want to write a
+   * Keybase bot that talks shit at anyone who dares approach it, this is the
+   * function to use.*
+   *
+   * Specifically, it will call the `onMessage` function you provide for every
+   * message your bot receives.*
+   *
+   *  @memberof Bot
+   *
+   *  @example
```

Comment:
I updated this example in #33, I can come back and update this once we merge that in.